tests/test_simplates.py

```diff
@@ -38,14 +38,12 @@ def test_cant_implicitly_override_state(harness):
 
 
 def test_can_explicitly_override_state(harness):
-    output = harness.simple("[---]\n"
-        "class Output: body=media_type=charset=code=299\n"
-        "state['output'] = Output()\n"
-        "[---]\n"
-        "bar"
```

Comment:
I was originally trying to remove `output` from the context entirely, but it would have required modifying the renderers, so I stopped halfway: the simplate (and renderers) can modify `output`, but not replace it with another object. Now that you mention it I'm not sure it's really better this way. What do you think?